### DISCUSSION TITLE
Bienvenido al Foro de DBV Combinar Correspondencia

### DISCUSSION THREAD

**Discussion by davidbuenov** [davidbuenov/CombinarCorrespondencia#1] [General]
## 👋 ¡Bienvenidos!

La idea de este foro es la de unir a todos los interesados en este proyecto para formar una comunidad y ayudar a que el proyecto pueda mejorar cada vez más hasta llegar a ser realmente la solución definitiva para combinar correspondencia

  Se espera que:
  * Hagas los comentarios que creas interesantes relacionados con el tema
  * Compartas tus ideas de cómo podría mejorarse.
  * Ayudes a dar difusión en tus redes del proyecto para que llegue a más usuarios.
  * Comentes los problemas que hayas podido tener y si lo has resuelto para que cuentes cómo
  * Aportar nuevas soluciones en Microsoft Word y VBA que puedan mejorar los resultados
  * Estamos abiertos todas las nuevas ideas para construir una potente comunidad. 💪.

Para comenzar, tu comentario, preséntate y dinos cual es tu vinculación con la combinación de correspondencia, VBA o Word avanzado.

**Comment by lazarolazarte** [comment]
Hola David, saludos desde argentina.
Te cuento estoy generando boletines en una planilla de word, tomando datos de planilla excel . Hago combinación de correspondencia y del archivo generado aplico el DVB Menu. y cuando quiero generar los pdf y que se manden por mail a los docentes que le correspondan me sale el siguiente error que no puedo resolver.
"EDBV. La carpeta no existe o se está intentando generar un archivo que ya está abierto"
Alguna sugerencia o ayuda?

**Reply by reynerth** [reply]
a mi me paso algo similar, tiene que ver con la carpeta predeterminada D:\temp\misdocs específicamente el problema es que o la carpeta no existe (porque puede que no tengas un disco D) o que por alguna razón referente a permisos no tengas la capacidad de grabar en dicha carpeta. recomiendo revisar que tengas permisos y que la carpeta en cuestión exista, o simplemente elegir otra carpeta para guardar la salida de la macro...

Se que es un tema muerto de hace 3 años pero espero que esto le pueda servir a alguien

**Reply by davidbuenov** [reply]
Gracias por tu aportación, aceptado el pull request.

**Comment by davidbuenov** [comment]
Hola,

Espero que estés usando la última versión de combinar correspondencia. Eso
te puede salir por:
1.- No estes usando las etiquetas correctamente, mira la documentación, ya
no se usa _NOMBRE-  pues daba problema con los guiones (míralo aquí:
https://github.com/davidbuenov/CombinarCorrespondencia/blob/main/Ayuda.md )
2. Has generado el primer documento pdf y lo has abierto, lo que hace que
el documento esté bloqueado y no se pueda escribir (cierra el lector de
pdfs que estés
 usando)
3.- Que hayas dejado la carpeta por defecto que estaba en d:\temp y no
tengas esa unidad.

Suerte, y si todo va bien, me gustaría poder añadir a tu
empresa/organización  a la lista de usuarios
https://github.com/davidbuenov/CombinarCorrespondencia/blob/main/Opiniones.md

El mié, 6 oct 2021 a las 22:16, lazarolazarte ***@***.***>)
escribió:

> Hola David, saludos desde argentina.
> Te cuento estoy generando boletines en una planilla de word, tomando datos
> de planilla excel . Hago combinación de correspondencia y del archivo
> generado aplico el DVB Menu. y cuando quiero generar los pdf y que se
> manden por mail a los docentes que le correspondan me sale el siguiente
> error que no puedo resolver.
> "EDBV. La carpeta no existe o se está intentando generar un archivo que ya
> está abierto"
> Alguna sugerencia o ayuda?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/davidbuenov/CombinarCorrespondencia/discussions/1#discussioncomment-1438731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD4BN6H7U6E3SASTXEAF7DLUFSVB3ANCNFSM4UUEOBZQ>
> .
>


-- 

David Bueno Vallejo

"Vive como si fueras a morir mañana. Aprende como si fueras a vivir para
siempre." - Gandhi

**Reply by Moises-AH** [reply]
Hola David buenos días, me sucede la misma incidencia, ya verifique la ruta y que no este abierto algún archivo pero me sigue apareciendo el mismo mensaje, me podrías orientar por favor.
![image](https://user-images.githubusercontent.com/95299281/144074842-fe09999b-dfc4-47f4-8951-096ba5ef1c93.png)

**Comment by Caponsito** [comment]
Hola David, aunque el tema lleve tiempo parado un tiempo, me gustaría preguntarte si hay manera de modificar la carpeta de guardado por defecto (D:\temp\misdocs) por otra que yo pudiera elegir. 

Gracias por tu enorme trabajo
